src/mixins/common-mixin.ts

```diff
@@ -29,6 +31,21 @@ export default class CommonMixin extends Vue {
     return (this.entityType === EntityTypes.CORP)
   }
 
+  get corpDisplayName (): string {
```

Comment:
Come to think of it, this could have been a global getter.
But it's OK here too.